config.json

```diff
@@ -15,18 +15,10 @@
     "indent_size": 4
   },
   "files": {
-    "solution": [
-      "source/%{pascal_slug}.ceylon"
-    ],
-    "test": [
-      "source/%{pascal_slug}Test.ceylon"
-    ],
-    "example": [
-      "example/%{pascal_slug}.ceylon"
-    ],
-    "exemplar": [
-      ".meta/Exemplar.ceylon"
-    ]
+    "solution": [],
+    "test": [],
+    "example": [],
+    "exemplar": []
```

Comment:
I can't make any generalized assumptions on the file names, so for now I've left them empty.